packages/plugin-commands-outdated/src/outdated.ts

```diff
@@ -244,6 +250,37 @@ ${renderCurrent(outdatedPkg)} ${chalk.grey('=>')} ${renderLatest(outdatedPkg)}`
     .join('\n\n') + '\n'
 }
 
+function renderOutdatedJSON (outdatedPackages: readonly OutdatedPackage[], opts: { long?: boolean }) {
+  const outdatedPackagesJSON = {} as {
```

Comment:
I wondered whether I should create a function that is almost equivalent to `renderLatest`, only without the output with chalk, but I was concerned that the implementation would be scattered, so I implemented it by adding `withoutChalk` argument which disables chalk output